src/cmd/tools/m3ctl/yaml/examples/newNode.yaml

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
Can you rename this filename to `new_node.yaml`?  All our file names are lower case under score separated.